mlflow/store/model_registry/sqlalchemy_store.py

```diff
@@ -358,7 +360,8 @@ def _parse_search_registered_models_order_by(cls, order_by_list):
                 else:
                     clauses.append(field.desc())
 
-        clauses.append(SqlRegisteredModel.name.asc())
+        if not contains_name:
```

Comment:
I think it'd be a better approach to just verify that `clauses` doesn't contain 2 of the same field. In a similar vein, I think we probably don't want two clauses with the same field with both `ASC` and `DESC`